playbooks/packstack.yaml

```diff
@@ -1,13 +1,16 @@
 ---
 - hosts: packstack
+  gather_facts: yes
   become: yes
   become_method: sudo
   tasks:
     - name: Set server hostname
       hostname: name="{{ server_fqdn }}"
 
     - name: Install updates
-      yum: name=* state=latest update_cache=yes
+      package: name=* state=latest update_cache=yes
```

Comment:
nice - use the `package` module so this is more OS-independent. But keep the `var: value` style